lessons/Early-Math/associative-property/langs/en-us/pages/props-make-comp-easier.adoc

```diff
@@ -0,0 +1,101 @@
+= Properties Make Computation Easier
+
+Apply the Associative Property to draw the Circle of Evaluation that will make the computation the simplest. Evaluate the expression. The first one is done for you.
+
+++++
+<style>
+  body.workbookpage table.FillVerticalSpace {
+    grid-template-rows: unset !important; grid-auto-rows: 1fr;
+  }
+  div.circleevalsexp .value,
+  div.circleevalsexp .studentBlockAnswerFilled { min-width:unset; }
+</style>
+++++
+
+[.FillVerticalSpace, cols="^4a,^4a"]
+|===
+
+
+| @math{13 + 4 + 7 + 6}
+
+@vspace{2ex}
+
+@show{(coe '(+ (+ 13 7) (+ 4 6)))}
+
+@vspace{2ex}
+
+__30__
```

Comment:
Strongly recommend showing 20 + 10 before you get to 30. It will help some students recognize how the groupings simplify things... and, it's a step I would want my students to show in subsequent problems!